addr.go

```diff
@@ -34,6 +41,30 @@ func ParseAddr(s string) (*Addr, error) {
 	return &Addr{IPNet: m, Label: label}, nil
 }
 
+// ParseAddr parses the string representation of an address in the form $ip/$netmask.
+func ParseAddr(s string) (*Addr, error) {
+	m, err := ParseIPNet(s)
+	if err != nil {
+		return nil, err
+	}
+	return &Addr{IPNet: m}, nil
+}
```

Comment:
For example valid network address for interface "eth0" , user may enter:
10.16.0.1/32 xxx
And netlink.ParseAddr would worked fine, but AddrAdd will not. And there is need for just valid network address only